### ISSUE TITLE
Request to bypass https://anydebrid.com/

### ISSUE THREAD

**Issue by MDAtikurRahman181-15-2024** [Sainan/Universal-Bypass#1963]

**Comment by impulsivus** [comment]
The website is scam.

**Comment by impulsivus** [comment]
> There is nothing to bypass, this is 100% confirmed scam: [DRSDavidSoft/additional-hosts@ef03762](https://github.com/DRSDavidSoft/additional-hosts/commit/ef037625a6e2fee7eccb722b453ef3750b7f03c6)
> 
> This site only scrapes the filename from download hosts, and displays it alongside ads and fake timers to the user. It is also tracking events and sending them to 3rd party trackers (e.g. Yandex)
> 
> After several redirects to fake pages, it redirects to a final fake page where either a generic "Failed" message is shown, or the site urges the user ti try the fake "Premium" download mode.
> 
> All of the links are fake, and they don't go to anywhere useful, just ads, and tracking. This site should be blocked.

i have to disagree with you on some parts. yes, it's a scam, but i was able to download a few files and they were filled with just random 1kb gibberish. i wonder why they waste their horsepower on such a b.s. 

![image](https://user-images.githubusercontent.com/20463920/137587128-8c07c0ee-5b39-4f40-9970-7377e891ec8c.png)

here's an example from one of my downloads. file sizes match exactly, but they are nothing but fillers.
